src/pages/Search/Search.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+import { SearchBar, Loader, VideoDetailsCard } from '../../components';
+import { useVideosData } from '../../contexts';
+import { useAxios } from '../../hooks';
+import { VIDEOS_ACTIONS } from '../../utils';
+import { filterVideosBySearchQuery } from '../../utils';
+
+export const Search = () => {
+  const [searchQuery, setSearchQuery] = useState('');
```

Comment:
Suggestion: You can move the component to logic to a separate Container like SearchContainer and use it in the View component.